deeplake/core/vectorstore/deeplake_vectorstore.py

```diff
@@ -157,7 +159,11 @@ def add(
         embedding_data: Optional[Union[List, List[List]]] = None,
         embedding_tensor: Optional[Union[str, List[str]]] = None,
         return_ids: bool = False,
-        ingestion_batch_size: Optional[int] = None,
+        rate_limiter: Dict = {
+            "turn_on": False,
```

Comment:
Is something like "enabled" a better/more common key than "turn_on"?

Reply:
make sense